src/main/scala/com/atomist/rug/runtime/CommandHandler.scala

```diff
@@ -3,11 +3,12 @@ package com.atomist.rug.runtime
 import com.atomist.param.ParameterValues
 import com.atomist.rug.runtime.js.RugContext
 import com.atomist.rug.spi.Handlers.Plan
+import com.atomist.rug.spi.SecretAwareRug
 
 /**
   * For bot commands
   */
-trait CommandHandler extends ParameterizedRug {
+trait CommandHandler extends SecretAwareRug {
```

Comment:
So CommandHandler now extends SecretAwareRug, but SecretAwareRug does not extend ParameterizedRug. Is anything lost there?